lib/porky_lib/symmetric.rb

```diff
@@ -61,56 +62,99 @@ def decrypt_data_encryption_key(ciphertext_key, encryption_context = nil)
     resp.plaintext
   end
 
-  def encrypt(data, cmk_key_id, ciphertext_dek = nil, encryption_context = nil)
+  def encrypt(data, cmk_key_id, ciphertext_dek = nil, encryption_context = nil, track_statistics = false)
```

Comment:
If you modify the signature to use hash options, i.e. `def encrypt(data, cmk_key_id, ciphertext_dek = nil, encryption_context = nil, track_statistics: false)` you can avoid the extra `nils` in your methods calls, and it may make it a bit easier to read: 
`symmetric.encrypt(plaintext_data, default_key_id, track_statistics: true)`